### ISSUE TITLE
conf Loss seems wrong

### ISSUE THREAD

**Issue by jediofgever** [Yuanchu/YOLO3D#3]
Hi, 
So I trained the network I was checking the logs with tensorboard, I come up to see the values of conf loss seems irrelevant. In below picture you can see 
![image](https://user-images.githubusercontent.com/18460336/50195904-04086d80-037c-11e9-8989-e6e3cc608579.png)
Did you have this issue ?

**Comment by Yuanchu** [comment]
This matches our observation as well. Might have to do with hyper-parameters and learning rate. Feel free to request a merge if you can fix it. Thanks in advance!

**Comment by abhigoku10** [comment]
@Yuanchu @jediofgever  where you able to fix this issue , i am in need to fix this section
